operators/test/e2e/smoke_test.go

```diff
@@ -31,14 +31,20 @@ func TestSmoke(t *testing.T) {
 	test.ExitOnErr(decoder.Decode(&apmBuilder.ApmServer))
 	test.ExitOnErr(decoder.Decode(&kbBuilder.Kibana))
 
+	ns := test.Ctx().ManagedNamespace(0)
 	esBuilder = esBuilder.
-		WithNamespace(test.Ctx().ManagedNamespace(0)).
+		WithRandomPrefixName().
```

Comment:
In cases like this where a single test creates multiple objects, would it make sense to share the random suffix between all of them so that the relationship is apparent and inspecting the objects with kubectl is easier?

Reply:
It makes sense. It looks like a small optimization for human debug needs that does not cost a lot.